src/node/export.ts

```diff
@@ -119,11 +121,15 @@ async function copyFootage(footage: Footage[], projectDir: string) {
  *
  * @param zipPath The path of the directory to zip.
  */
-function zip(zipPath: string): Promise<void> {
+function zip(zipPath: string, projectName: string): Promise<void> {
```

Comment:
* `zipPath` is also wrong, should be `targetPath` or something like that
* can you please go over naming here so other people would know what's going on when they are looking at the code